pyanaconda/modules/timezone/timezone.py

```diff
@@ -101,7 +101,7 @@ def process_kickstart(self, data):
                 source.type = TIME_SOURCE_POOL
                 source.hostname = source_data.ntp_pool
             else:
-                KickstartParseError(
+                raise KickstartParseError(
```

Comment:
I don't know if we shouldn't delete this part. We are not able to test it.

Reply:
I didn't look that far, just wanted to make tests pass...